assets/javascripts/kitten/components/molecules/buttons/arrow-container-button/stories.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { ArrowContainerButton } from './index'
+
+const args = {
+  iconTitle: 'Carte cadeau',
+}
+
+const argsTypes = {
+  iconTitle: {
+    name: 'iconTitle',
+    control: 'string',
```

Comment:
je crois que c'est `text` et non `string`

```suggestion
    control: 'text',
```